iothub/device/src/InternalClient.cs

```diff
@@ -109,6 +109,7 @@ internal class InternalClient : IDisposable
         private readonly SemaphoreSlim _methodsDictionarySemaphore = new SemaphoreSlim(1, 1);
         private readonly SemaphoreSlim _deviceReceiveMessageSemaphore = new SemaphoreSlim(1, 1);
         private readonly SemaphoreSlim _moduleReceiveMessageSemaphore = new SemaphoreSlim(1, 1);
+        private readonly SemaphoreSlim _twinDesiredPropertySemaphore = new SemaphoreSlim(1, 1);
```

Comment:
Where are these being disposed?

Reply:
Taking care of this now, thanks for noticing this.